sdk/search/Azure.Search.Documents/src/Generated/DocumentsRestClient.cs

```diff
@@ -207,7 +207,7 @@ internal HttpMessage CreateGetRequest(string key, IEnumerable<string> selectedFi
         /// <param name="selectedFields"> List of field names to retrieve for the document; Any field not retrieved will be missing from the returned document. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="key"/> is null. </exception>
-        public async Task<Response<IReadOnlyDictionary<string, object>>> GetAsync(string key, IEnumerable<string> selectedFields = null, CancellationToken cancellationToken = default)
+        public async Task<Response<object>> GetAsync(string key, IEnumerable<string> selectedFields = null, CancellationToken cancellationToken = default)
```

Comment:
Search changes look good but why are we adding [CodeModel.yaml](https://github.com/Azure/azure-sdk-for-net/blob/1c6b2a529b9ef979641386c64e1a42a73d9bb6cd/sdk/search/Azure.Search.Documents/src/Generated/CodeModel.yaml) file for Search?

Reply:
Just for investigate. I'll remove.